test/integration/centos/inspec/centos_spec.rb

```diff
@@ -0,0 +1,71 @@
+# Test for the main configuration file ('/etc/yum.conf'cookstyle)
+describe file('/etc/yum.conf') do
```

Comment:
Please use the [ini](https://docs.chef.io/inspec/resources/ini/) resource instead.